### DISCUSSION TITLE
How to Reduce Model Size After Training?

### DISCUSSION THREAD

**Discussion by MikuAuahDark** [voicepaw/so-vits-svc-fork#398] [Q&A] [answered]
Can I reduce the model size from 500MB to less size, or is it not possible?

**Comment by xieyumc** [comment]
maybe it is not possible and actually not necessary(you can delete old model by set the config file)

**Comment by 34j** [comment, marked as answer]
I suppose it could be done by implementing pruning, but I don't see much benefit.
